modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -157,6 +157,20 @@ const Modal = ({
 		);
 	};
 
+	const StatusMessage = () => {
+		const [showMessage, setShowMessage] = useState(true);
+
+		useEffect(() => {
+			setTimeout(() => setShowMessage(false), 1000);
+		});
+
+		return showMessage ? (
+			<span className="sr-only" role="status">
+				{Liferay.Language.get('loaded')}
+			</span>
+		) : null;
+	};
```

Comment:
This component should be in root of the file, instead of defined as an inner component in `Modal`. [See here in details why](https://react.dev/learn/your-first-component#nesting-and-organizing-components).